core-meetings/2018-09-metro-meeting.md

```diff
@@ -0,0 +1,35 @@
+# Metro configurations changes: September 2018 Core Meeting
+
+As stewards of React Native engineers worldwide, we've contemplated ways to improve alignment and visibility of the framework's direction, where community members and Facebook alike joined to discuss issues and propose responses.
```

Comment:
Haha, this is such a nice phrase

Reply:
I think it was @turnrye that wrote it in the first meeting notes, I just keep copypasting it because I think it's important